bogota-july-2018/Dominik Ortiz Galop/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="author" content="Dominik Ortiz">
+    <title>Calculator</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <header>
+        <h1>Calculate something</h1>
+    </header>
+    <main>
+        <div id="container" class="container">
+            <div id="display" class="display"></div>
+            <button class="btn ">AC</button>
```

Comment:
You have unnecessary spaces after `btn` class

Reply:
typo error :/